tests/countries/test_singapore.py

```
@@ -194,6 +194,24 @@ def test_2025(self):
             ("2025-12-25", "Christmas Day"),
         )
 
+    def test_2026(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)